packages/billing/src/utils/paymentSplitter.ts

```diff
@@ -1,50 +1,54 @@
-const { Big } = require('big.js')
-const { get, set, omit } = require('lodash')
-
-/**
- * @typedef {Object} TRecipient
- * @property {string} bankAccount
- * @property {string} [bankName]
- * @property {string} bic
- * @property {string} [classificationCode]
- * @property {string} [iec]
- * @property {string} [name]
- * @property {string} [offsettingAccount]
- * @property {string} [territoryCode]
- * @property {string} tin
- */
+import { Big } from 'big.js'
+import get from 'lodash/get'
+import omit from 'lodash/omit'
+import set from 'lodash/set'
+
+export type TRecipient = {
```

Comment:
I thought we've finalised decision and T-typing

Reply:
Refactored